Nagstamon/QUI/settings_main.py

```diff
@@ -1642,3 +1646,13 @@ def retranslateUi(self, settings_main):
         self.label_acknowledge_submit_result.setTitle(_translate("settings_main", "Submit check result:"))
         self.label_defaults_submit_check_result_comment.setText(_translate("settings_main", "Comment:"))
         self.tabs.setTabText(self.tabs.indexOf(self.tab_defaults), _translate("settings_main", "Defaults"))
+
+
+if __name__ == "__main__":
```

Comment:
Where does this condition come from? Is it inserted by the GUI compiler pyuic5?

Reply:
Exactly